audio/voices.yaml

```diff
@@ -1,28 +1,26 @@
-AlliedEngineerVoice:
-	DefaultVariant: .wav
-	Voices:
-		Select: ienasea, ienaseb, ienasec, ienased
-		Move: ienamoa, ienamob, ienamoc
-		Action: ienaata, ienaatb, ienaatc
-		Feedback: ienafea, ienafeb, ienafec
-		Die: ienadia, ienadib, ienadic, ienadid
+EngineerVoice:
+	DefaultVariant: .wav
+	Prefixes:
```

Comment:
Sure you can. It just requires slightly different setup in your sequences
file.
On Dec 22, 2015 3:44 PM, "Iamgoofball" <notifications@github.com> wrote:

> In audio/voices.yaml
> <https://github.com/OpenRA/ra2/pull/1#discussion_r48217184>:
>
> > @@ -1,28 +1,26 @@
> > -AlliedEngineerVoice:
> > -	DefaultVariant: .wav
> > -	Voices:
> > -		Select: ienasea, ienaseb, ienasec, ienased
> > -		Move: ienamoa, ienamob, ienamoc
> > -		Action: ienaata, ienaatb, ienaatc
> > -		Feedback: ienafea, ienafeb, ienafec
> > -		Die: ienadia, ienadib, ienadic, ienadid
> > +EngineerVoice:
> > +	DefaultVariant: .wav
> > +	Prefixes:
>
> You can't have two infantry units with different engine names using the
> same shp. I tried. It didn't work, we'd need to ship sovietengineer.shp and
> alliedengineer.shp alongside the mod, and that'd be distributing EA assets
> illegally.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenRA/ra2/pull/1/files#r48217184>.
>